pages/api/og.tsx

```diff
@@ -0,0 +1,82 @@
+import { ImageResponse } from '@vercel/og';
+import { NextRequest } from 'next/server';
+
+export const config = {
+  runtime: 'edge',
+};
+
+const font = fetch(
+  new URL('../../public/fonts/Poppins-Medium.ttf', import.meta.url),
+).then((res) => res.arrayBuffer());
+
+export default async function handler(req: NextRequest) {
+  const { searchParams } = req.nextUrl;
+  const title = searchParams.get('title');
+  const category = searchParams.get('category');
+  const fontData = await font;
+
+  return new ImageResponse(
+    (
+      <div
+        style={{
+          height: '100%',
+          width: '100%',
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          justifyContent: 'center',
+          backgroundImage:
+            'url(https://res.cloudinary.com/doxkn6teg/image/upload/v1677773730/bg_ixxj23.jpg)',
+          paddingLeft: 135,
+          paddingRight: 450,
+        }}
+      >
+        <div
+          style={{
+            display: 'flex',
+            fontSize: 90,
+            fontFamily: 'Poppins',
+            letterSpacing: '-0.05em',
+            fontStyle: 'normal',
+            color: 'white',
+            lineHeight: '90px',
+            whiteSpace: 'pre-wrap',
+          }}
+        >
+          {title}
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            fontSize: 30,
+            fontFamily: 'Poppins',
```

Comment:
I think we use Inter across the brand but @adamsoffer may know more

Reply:
Yup, please change to Inter.